src/main/scala/com/github/fulrich/testcharged/generators/ids/UuidGenerators.scala

```diff
@@ -0,0 +1,9 @@
+package com.github.fulrich.testcharged.generators.ids
+
+import java.util.UUID
+
+import org.scalacheck.Gen
+
+object UuidGenerators {
+  val uuid: Gen[UUID] = Gen.uuid
```

Comment:
I would suggest naming this default to keep with the conventions of the library. It will also let you create a typeclass later extending from `DefaultCaller` which will call default for you to avoid needing to constantly retype it.

Reply:
I don’t understand, the DSL is `Generate.uuid.value`
if I name it as default, how will the DSL work?
`Generate.default.value`?

Reply:
done